### PR TITLE
Add certificateUpload to api documentation

### LINKED ISSUE #311: Certificate POST is undocumented
The new certificate upload handler should be documented in `HTTP_API.md`

### PR DESCRIPTION
**Pull request by Alexjsenn**
Fixes #311

### CODE REVIEW

File: HTTP_API.md [hunk 1]
```diff
@@ -1217,3 +1219,30 @@
...
+
+* #### `CertificatePostHandler`
+
+    ###### synopsis
+    Uploads an SSL Certificate from the client, and saves it to the truststore directory.
+
+    ###### request
+    `POST /api/v2/certificates`
+
+    The certificate must be DER-encoded and can be either binary of base64. The supported extensions are .der, .cer, .pem.
```
Comment (andrewazores) [inline HTTP_API.md:1233]: typo: "can be either binary *of* base64"

Comment (Alexjsenn) [reply HTTP_API.md]: My bad

**Comment by andrewazores** [comment]
Looks okay to me. @vic-ma ?

**Review by vic-ma — COMMENTED** [review]
Looks good otherwise!

File: HTTP_API.md [hunk 2]
```diff
@@ -1217,3 +1219,30 @@
...
+    Uploads an SSL Certificate from the client, and saves it to the truststore directory.
+
+    ###### request
+    `POST /api/v2/certificates`
+
+    The certificate must be DER-encoded and can be either binary or base64. The supported extensions are .der, .cer, .pem. 
+    The certificate should be uploaded in a form with the name `cert`.
+
+    ###### response
+    `200` - The body is `saved: ` followed by the path of the saved file.
```
[the remark this answers is not in the document]
Comment (Alexjsenn) [reply HTTP_API.md]: Ah, makes sense!

File: HTTP_API.md [hunk 3]
```diff
@@ -1217,3 +1219,31 @@
...
+    Uploads an SSL Certificate from the client, and saves it to the truststore directory.
+
+    ###### request
+    `POST /api/v2/certificates`
+
+    The certificate must be DER-encoded and can be either binary or base64. The supported extensions are .der, .cer, .pem. 
+    The certificate should be uploaded in a form with the name `cert`.
+
+    ###### response
+    `200` - The body is `saved: $PATH`, where `$PATH` is the path of the saved file.
```
Comment (vic-ma) [inline HTTP_API.md:1237]: Ah, one last thing, it looks like the `saved:` here should be capitalized.

**Review by vic-ma — APPROVED** [review]

**Pull request merged**